### ISSUE TITLE
Cannot assign value of type 'ViewController' to type 'UITextFieldDelegate?'

### ISSUE THREAD

**Issue by takasurazeem** [chronotruck/FlagPhoneNumber#58]
**Describe the bug**
Delegate method is not getting called, Insert ' as! UITextFieldDelegate'
**To Reproduce**
```class ViewController: UIViewController, FPNTextFieldDelegate {```
In viewDidLoad
```phoneNumberTextField.delegate = self```

**Expected behavior**
The delegate should be set and methods may get called.
**Screenshots**
![image](https://user-images.githubusercontent.com/2876680/50536436-6d576000-0b76-11e9-83eb-499f2a5f77d7.png)

**Smartphone (please complete the following information):**
 - Device: [Simulator]
 - OS: [e.g. Xcode 10 iOS 12.1]
 - Version [Swift 4.2]
 
 **FlagPhoneNumber (please complete the following information):**
 - Version [Master]

**Additional context**
N/A

**Comment by grifas** [comment]
You didn't add the `UITextFieldDelegate` to your ViewController class so it's why there is an error.

Maybe `FPNTextFieldDelegate` should be inherited from `UITextFieldDelegate` like:

```
public protocol FPNTextFieldDelegate: UITextFieldDelegate {
	func fpnDidSelectCountry(name: String, dialCode: String, code: String)
	func fpnDidValidatePhoneNumber(textField: FPNTextField, isValid: Bool)
}
```

I pushed a fix on `dev` branch.

**Comment by grifas** [comment]
@takasurazeem It's now fixed on `0.7.2`.
